src/app/routes/paths.json

```diff
@@ -234,13 +234,13 @@
       "exact": false
     },
     {
-      "id": "share-photos-token",
+      "id": "share-photos-token-2",
       "layout": "share",
       "path": "/d/sh/photos/:token([a-z0-9]{24})/:code",
       "exact": false
     },
     {
-      "id": "share-photos-token",
+      "id": "share-photos-token-3",
```

Comment:
I would remove all the /photos/ as those shares no longer exist

Reply:
Don't we leave it to support any photos links that may still exist?